ear/fileio/adm/elements/main_elements.py

```diff
@@ -195,12 +285,28 @@ def add_encodePackFormat(decode_pack, new_encode_pack):
 
 @attrs(slots=True)
 class Frequency(object):
+    """ADM frequency element
+
+    Attributes:
+        lowPass (Optional[float])
+        highPass (Optional[float])
+    """
+
     lowPass = attrib(default=None, validator=optional(instance_of(float)))
     highPass = attrib(default=None, validator=optional(instance_of(float)))
 
 
 @attrs(slots=True)
 class AudioChannelFormat(ADMElement):
+    """ADM audioChannelFormat
+
+    Attributes:
+        audioChannelFormatName (str)
+        type (TypeDefinition): ``typeDefintion`` and/or ``typeLabel``
+        audioBlockFormats (list[AudioBlockFormat])
```

Comment:
I don't know whether this is the place, but in the docs generated for the AudioChannelFormat class, the entry for https://ear.readthedocs.io/en/docs/ADM_data.html#ear.fileio.adm.elements.AudioChannelFormat does not have a link in place for AudioBlockFormat in  "list[AudioBlockFormat]".  It has a link to the basic "list" type, only.

Reply:
I fixed this a while ago (sorry) -- turns out the type was originally called `BlockFormat` not `AudioBlockFormat` for some reason.